### DISCUSSION TITLE
Markdown tables isn't formatted in newer Quarto versions

### DISCUSSION THREAD

**Discussion by snhansen** [quarto-dev/quarto-cli#10902] [Q&A] [answered]
### Description

I just noticed that markdown tables suddenly doesn't have any formatting done. Is this intentional? The reason I'm asking is because it's how (most of the) tables are presented in the documentation, see e.g. https://quarto.org/docs/authoring/tables.html#markdown-tables

This example from the documentation

```
| Col1 | Col2 | Col3 |
|------|------|------|
| A    | B    | C    |
| E    | F    | G    |
| A    | G    | G    |

: My Caption {#tbl-letters}

See @tbl-letters.
```

renders like this for me:

![image](https://github.com/user-attachments/assets/193ad76c-8d9b-46b9-9f79-a3ec7b1c22c2)

**Comment by cderv** [comment]
Which version of Quarto are you using ? Is this pre-release ? 

I do think there was some change in Pandoc recently with table class removed. We may need to adapt to it differently. 

Please can you share you quarto version ? thank you !

**Reply by snhansen** [reply]
Yes, of course! It's 1.6.14, so this is pre-release.

**Reply by cderv** [reply]
Ok so could definitely be linked to this. Thanks a lot !

**Reply by cderv** [reply, marked as answer]
Following up on 
- https://github.com/quarto-dev/quarto-cli/issues/10904
